core/src/main/java/com/exadel/etoolbox/linkinspector/core/services/helpers/CsvHelper.java

```diff
@@ -0,0 +1,36 @@
+package com.exadel.etoolbox.linkinspector.core.services.helpers;
+
+import com.exadel.etoolbox.linkinspector.core.models.ui.GridViewItem;
+import com.exadel.etoolbox.linkinspector.core.services.data.models.GridResource;
+import org.apache.sling.api.resource.ResourceResolver;
+
+import java.util.List;
+
+public interface CsvHelper {
+
+    /**
+     * Reading items from csv and adapt them to the {@link GridResource}
```

Comment:
(grammar) "Reads... and adapts...". 
(spelling) (Here and below in the whole PR). CSV is an abbreviation and must be spelled in capital letters. Same is "JCR",